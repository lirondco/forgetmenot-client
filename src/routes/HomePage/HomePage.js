import React, { Component } from 'react'
import TokenService from '../../services/token-service'
import { Section } from '../../components/Utils/Utils'
import Home from '../../components/Home/Home'
import ListsPage from '../ListsPage/ListsPage'

export default class HomePage extends Component {
  render() {
    return (
    <Section className = "homepage">
        {TokenService.hasAuthToken()
          ? <ListsPage />
          : <Home /> }
    </Section>
    )
  }
}