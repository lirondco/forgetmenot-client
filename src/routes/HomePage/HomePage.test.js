import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import HomePage from './HomePage'

describe(`HomePage component`, () => {
  it('renders the home page', () => {
    const wrapper = shallow(<HomePage />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})