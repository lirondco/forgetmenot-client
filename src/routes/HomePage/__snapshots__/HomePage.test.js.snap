// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component renders the home page 1`] = `
<Section
  className="homepage"
>
  <Home />
</Section>
`;
